test/RelayHubGasCalculations.test.js

```diff
@@ -122,12 +122,12 @@ contract('RelayHub gas calculations', async function ([_, relayOwner, relayAddre
 
         // Magic numbers seem to be gas spent on calldata. I don't know of a way to calculate them conveniently.
         await expectEvent.inTransaction(tx, TestSponsorVariableGasLimits, 'SampleRecipientPreCallWithValues', {
-          gasleft: (gasData.preRelayCallGasLimit - magicNumbers.pre).toString(),
+          gasleft: (gasData.preRelayedCallGasLimit - magicNumbers.pre).toString(),
           arcGasleft: (gasData.acceptRelayedCallGasLimit - magicNumbers.arc).toString(),
           maxPossibleGas: gasData.maxPossibleGas.toString()
         })
         await expectEvent.inTransaction(tx, TestSponsorVariableGasLimits, 'SampleRecipientPostCallWithValues', {
-          gasleft: (gasData.postRelayCallGasLimit - magicNumbers.post).toString()
+          gasleft: (gasData.postRelayedCallGasLimit - magicNumbers.post).toString()
```

Comment:
sorry to nag on unrelated issues, but such "magic numbers" should *at least* be used in several separate tests, which takes different params.
Otherwise, this test is meaningless

Reply:
Well, this test is semi-meaningless. I want the value to be 50'000, but instead, it is 49'123, so I know the magic 877 gas was used for calldata and stuff. If instead of 49'123 I one day get 12, of "potato", the test will fail and catch a regression.